### DISCUSSION TITLE
WARNING: Unable to resolve link target:

### DISCUSSION THREAD

**Discussion by ning3739** [quarto-dev/quarto-cli#3624] [Q&A] [answered]
I received the following message when publishing HTML to quarto pub:

```
output file: Data-visualization.knit.md
WARNING: Unable to resolve link target: series/Data Visualization/index.qmd
WARNING: Unable to resolve link target: series/Data Visualization/index.qmd
WARNING: Unable to resolve link target: series/Data Visualization/about.qmd

```
My website seems to be loading normally, but Rstudio keeps giving me warning messages when I update something to publish in quarto pub.

Has anyone encountered such a situation? What should I do? Thanks 🙏

**Comment by dragonstyle** [comment, marked as answer]
Usually this means that there are invalid links to  documents in your website. Are those links valid from all pages that you use them? Since the links don’t begin with a slash character, they will be relative to the document  which the links appear in…
